packages/core/package.json

```diff
@@ -2,32 +2,26 @@
   "name": "navi-core",
   "version": "0.0.0",
   "description": "Library of common logic, components, and styles for navi",
-  "directories": {
-    "test": "tests"
-  },
   "scripts": {
     "postinstall": "bower install",
     "build": "ember build",
-    "start": "ember server",
-    "test": "export COVERAGE=true; ember test --test-port 0"
+    "lint:hbs": "ember-template-lint .",
+    "lint:js": "eslint .",
+    "start": "ember serve",
+    "test": "ember test",
```

Comment:
If forget, Do we still need the `export COVERAGE=true`?

Reply:
These scripts aren't run in travis so we don't need coverage.